src/ServerConnectionAcceptor.cpp

```diff
@@ -159,26 +156,55 @@ void ServerConnectionAcceptor::processFrame(
     default: {
       transport->outputFrameOrEnqueue(
           frameSerializer->serializeOut(Frame_ERROR::unexpectedFrame()));
-      closeAndRemoveConnection(
-          connectionHandler,
-          std::move(transport),
-          std::runtime_error("invalid"));
+      error("Invalid frame received instead of SETUP/RESUME");
       break;
     }
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Let's remove them, they're existence is quite subjective and I personally don't feel like they add much.